Gemfile

```diff
@@ -6,6 +6,8 @@ gem 'ruby-fifo', github: 'shurizzle/ruby-fifo'
 
 group :development do
   gem 'rspec'
+  gem 'coveralls'
+  gem 'simplecov'  
```

Comment:
Also since they were added to gemspec the can be removed from Gemfile (except lines belongs to gem that referenced to github).btw, why do you need the rspec and rubocop?